### DISCUSSION TITLE
cursor does not blink when in copy-mode ?

### DISCUSSION THREAD

**Discussion by mckellygit** [wezterm/wezterm#1760] [Q&A] [answered]
Hi,
Thank you so much for wezterm.
When I go into wezterm copy-mode (or also tmux copy-mode) the blinking cursor stops blinking.
Is this configurable ?
thx,
-m

**Comment by wez** [comment, marked as answer]
This is deliberate (aa61a6b065fb2d9a0f15b32350a69b090b81d165) and it is not configurable
